cmake/modules/AddSwift.cmake

```diff
@@ -46,12 +46,12 @@ function(_add_variant_c_compile_link_flags
     "-target" "${SWIFT_SDK_${sdk}_ARCH_${arch}_TRIPLE}")
 
   list(APPEND result
-    "-isysroot" "${SWIFT_SDK_${sdk}_PATH}")
+    "-isysroot" "\"${SWIFT_SDK_${sdk}_PATH}\"")
```

Comment:
The CMake changes here seem odd to me. This is a list and these paths are already quoted, which means at this point in time CMake knows what one element consists of. Can we accomplish this using the `VERBATIM` setting for all `add_custom_command_target` calls instead?

(I'd be fine with forcing this setting in `add_custom_command_target` itself, if it doesn't break anything.)